src/Draco.ProjectFile/DracoBuildTask.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Build.Utilities;
+
+namespace Draco.ProjectFile;
+
+public class DracoBuildTask : ToolTask
+{
+    public override bool Execute()
+    {
+        var files = Directory.EnumerateFiles(this.ProjectDirectory, "*.draco", SearchOption.TopDirectoryOnly).Where(x => x == Path.Combine(this.ProjectDirectory, "main.draco"));
```

Comment:
Is there a reason we only search for top-level source files?

Reply:
Let's just keep `files` as a list, then have a `var mainFile = files.FirstOrDefault(f => ...)`